pages.th/common/clamdscan.md

```diff
@@ -0,0 +1,36 @@
+# clamdscan
+
+> โปรแกรมแสกนหาไวรัสบนคอมมานด์ไลน์ โดยใช้โปรแกรมพื้นหลัง ClamAV
+> ข้อมูลเพิ่มเติม: <https://www.clamav.net>.
+
+- แสกนข้อบกพร่องของไฟล์หรือไฟล์ในไดเรคทอรี
+
+`clamdscan {{path/to/file_or_directory}}`
```

Comment:
Translate this path, and other command arguments inside `{{}}` throughout the pages.

Reply:
Done.